UnityProject/Assets/Scripts/Core/Database/ServerData.ServerStatus.cs

```diff
@@ -96,9 +96,20 @@ IEnumerator SendServerStatus()
 				{
 					string s = req.GetResponseHeader("set-cookie");
 					hubCookie = s.Split(';') [0];
-					req = UnityWebRequest.Get("http://ipinfo.io/ip");
-					yield return req.SendWebRequest();
-					publicIP = Regex.Replace(req.downloadHandler.text, @"\t|\n|\r", "");
+					if (string.IsNullOrEmpty(config.PublicAddress) == false)
+					{
+						publicIP = config.PublicAddress;
+					} 
+					else if (string.IsNullOrEmpty(config.BindAddress) == false)
+					{
+						publicIP = config.BindAddress;
+					}
+					else
+					{
+						req = UnityWebRequest.Get("http://ipinfo.io/ip");
```

Comment:
I gave a quick look at that website and it looks like a paid service?

Reply:
Not my code

Reply:
The /ip endpoint just returns any public ip, and it's been in the code for a loong time now so I don't see any reason to change it unless there's a more guaranteed alternative.